XamlControlsGallery/ControlPages/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,66 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.CornerRadiusPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:AppUIBasics"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+    <StackPanel>
+        <TextBlock IsTextSelectionEnabled="True">
+            <LineBreak />
+            <Run Text="For basic UI elements like buttons that users see on screen at all times," />
+            <Run Text="the default radius value is" />
+            <Run FontWeight="bold" Text="2px" />
+            <Run Text="." />
+            <LineBreak />
+            <LineBreak />
+            <Run Text="For transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs," />
+            <LineBreak />
+            <Run Text="the default radius value we use for is" />
+            <Run FontWeight="Bold" Text="4px" />
+            <Run Text="." />
+        </TextBlock>
+        <local:ControlExample
+            x:Name="Example1"
+            HeaderText="A button with CornerRadius"
+            MinimumUniversalAPIContract="7">
+            <Button contract7Present:CornerRadius="5" Content="MyButton" />
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Button CornerRadius=&quot;5&quot; Content=&quot;MyButton&quot;/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample
+            x:Name="Example2"
+            HeaderText="A button without CornerRadius"
+            MinimumUniversalAPIContract="7">
+            <Button contract7Present:CornerRadius="0" Content="MyButton" />
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Button CornerRadius=&quot;0&quot; Content=&quot;MyButton&quot;/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample MinimumUniversalAPIContract="7">
+            <Button contract7Present:CornerRadius="5,0,5,0" Content="CornerRadius: 5,0,5,0" />
+        </local:ControlExample>
+        <local:ControlExample MinimumUniversalAPIContract="7">
+            <Button contract7Present:CornerRadius="5,5,0,0" Content="CornerRadius: 5,5,0,0" />
+        </local:ControlExample>
```

Comment:
Not sure if we those provide much value. Maybe we could show other controls that exercise the API?

Reply:
I'm not quite sure I understand your request. Could you please re-word it?

Reply:
Showing that you can set the value `5,0,5,0` but also the value `5,5,0,0` to the corner radius of a button does probably don't help much more than just showing one of those values. How about showing what setting the corner radius does on other controls such as TextBox or SplitButton?

Reply:
Okay, I'll do that. Thanks